### PR TITLE
Solving basic tasks (creating app & weather)

### PR DESCRIPTION
**Pull request by DariaaaP**
1) Created a new app with the output of a string with the current date;
2) Fixed the display of weather cards depending on different conditions.

### CODE REVIEW

**Comment by jsru-1** [comment]
Добавляю преподавателя (@ShGKme) для код-ревью.

File: 01-basics/10-create-app/script.js [hunk 1]
```diff
@@ -1 +1,19 @@
...
+    function formatAsLocalDate(timestamp) {
+      return new Date(timestamp).toLocaleString(navigator.language, {
+        dateStyle: 'long',
+      })
+    }
+
+    return { formatAsLocalDate }
+  },
+
+  template: `<div>Сегодня {{formatAsLocalDate(new Date())}}</div>`,
```
Comment (ShGKme) [inline 01-basics/10-create-app/script.js:16]: Text interpolation is always used with spaces: `{{ formatAsLocalDate(new Date()) }}`

**Review by ShGKme — APPROVED** [review]
Approved

File: 01-basics/10-create-app/script.js (hunk 1 above)
Comment (ShGKme) [reply 01-basics/10-create-app/script.js:16]: Same with the next problem

**Pull request merged**